homeassistant/components/unifi/unifi_client.py

```diff
@@ -5,7 +5,7 @@
 from .unifi_entity_base import UniFiBase
 
 
-class UniFiClient(UniFiBase):
+class UniFiClientBase(UniFiBase):
```

Comment:
`UniFiClientTracker` shouldn't need to override `device_info` to force it to `None`.
I have therefore created this intermediate class which is specific to `UniFiClientTracker`, whilst all other child classes still have the `device_info`.